tests/DevApps/MtlsPop/MtlsPopClient/Program.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Graph;
+using Microsoft.Identity.Abstractions;
+using Microsoft.Identity.Web;
+
+namespace MtlsPopSample
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            var tokenAcquirerFactory = TokenAcquirerFactory.GetDefaultInstance();
+
+            tokenAcquirerFactory.Services.AddLogging(builder =>
+                builder.AddConsole().SetMinimumLevel(LogLevel.Information));
+
+            tokenAcquirerFactory.Services.AddDownstreamApi("WebApi",
+                tokenAcquirerFactory.Configuration.GetSection("WebApi"));
+
+            tokenAcquirerFactory.Services.AddMicrosoftGraph();
+
+            var sp = tokenAcquirerFactory.Build();
+
+            Console.WriteLine("Scenario 1: calling web API with mTLS PoP token...");
+            var webApi = sp.GetRequiredService<IDownstreamApi>();
+            var result = await webApi.GetForAppAsync<IEnumerable<WeatherForecast>>("WebApi",
+                options =>
+                {
+                    options.AcquireTokenOptions.ExtraParameters ??= new Dictionary<string, object>();
+                    options.AcquireTokenOptions.ExtraParameters["RequestBoundToken"] = true; // mTLS PoP
```

Comment:
I thought we settled on using `ProtocolScheme` for allowing app developer to opt-in to this type of token. https://github.com/AzureAD/microsoft-identity-abstractions-for-dotnet/blob/35845ee8d662be389802d538802535a6d40607c0/src/Microsoft.Identity.Abstractions/DownstreamApi/AuthorizationHeaderProviderOptions.cs#L97